PerfectXL.EPPlus/VBA/ExcelVBAProject.cs

```diff
@@ -259,6 +259,10 @@ private void ReadProjectProperties()
                             _protection.VbeProtected = (cmg[0] & 4) != 0;
                             break;
                         case "DPB":
+                            if (!Regex.IsMatch(split[1], "^[\\dA-F]+$"))

```

Comment:
I think we can safely add `RegexOptions.IgnoreCase` here.

Reply:
True since [System.Globalization.NumberStyles.AllowHexSpecifier](https://github.com/PerfectXL/PerfectXL-EPPlus/blob/develop/PerfectXL.EPPlus/VBA/ExcelVBAProject.cs#L400) also allows [a-f]